packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -6,21 +6,23 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
-import TooltipPortalMounter from "./tooltip-portal-mounter.js";
+import {Text as WBText} from "@khanacademy/wonder-blocks-core";
+
 import ActiveTracker from "../util/active-tracker.js";
 import {
     TooltipAppearanceDelay,
     TooltipDisappearanceDelay,
 } from "../util/constants.js";
 
 import type {IActiveTrackerSubscriber} from "../util/active-tracker.js";
+import type {IIdentifierFactory} from "@khanacademy/wonder-blocks-core";
 
 type Props = {|
     /**
-     * A method that renders the content for anchoring the tooltip.
-     * This must return a TooltipPortalMounter component.
+     * The content for anchoring the tooltip.
+     * This component will be used to position the tooltip.
      */
-    children: (active: boolean) => React.Element<typeof TooltipPortalMounter>,
+    children: React.Element<any> | string,
```

Comment:
It took me a little while to understand this. It would be good to call out what we're looking for in the comment for this prop as we only want a React Element or a string (very explicitly) and a string is being wrapped with Text (which we should make clear in the documentation).

Reply:
A string can be wrapped int a `Text` node but other things can like booleans, arrays, etc.  I guess we could but it's probably a mistake to be passing those things as children.

Reply:
I'll add a comment.